cosmic_ray/operators/operator.py

```diff
@@ -53,3 +54,23 @@ def __repr__(self):
         return '{}({})'.format(
             self.__class__.__name__,
             ', '.join(args))
+
+
+class ReplacementOperator(Operator, metaclass=ABCMeta):
```

Comment:
Consider:

    from abc import ABC

then just inherit from ABC, which is just an empty class with ABCMeta as its metaclass, so

    class ReplacementOperator(Operator, ABC):